docs/api/navigators/TabNavigator.md

```diff
@@ -88,6 +88,7 @@ The route configs object is a mapping from route name to a route config, which t
 - `swipeEnabled` - whether to allow swiping between tabs
 - `animationEnabled` - whether to animate when changing tabs
 - `lazy` - whether to lazily render tabs as needed as opposed to rendering them upfront
+- `initialLayout` - optional object containing the initial `height` and `width`, can be passed to prevent the one frame delay in rendering
```

Comment:
I think it would be a good idea to link directly to react-native-tab-view here to provide more context to library users.

Reply:
Will try to update sometime tonight.